README.md

```diff
@@ -63,3 +63,14 @@ Now you'll see new sql files in `db/migration/migrations/sql/<timestamp>- add-ta
 You should edit the `<timestamp>-add-table-dbtable-up.sql` to contain your DDL statements.
 
 You could also add the corresponding `drop table` to `<timestamp>-add-table-dbtable-down.sql` if you ever want to undo migrations.
+
+## Run R Studio Server locally
+
```

Comment:
it would be nice also to have the instruction to install it.
Is Arena able to copy files in the docker dir?